packages/react/src/components/SideMenu/DragHandleMenu/DefaultItems/TableHeadersItem.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  BlockSchema,
+  DefaultBlockSchema,
+  DefaultInlineContentSchema,
+  DefaultStyleSchema,
+  InlineContentSchema,
+  SpecificBlock,
+  StyleSchema,
+} from "@blocknote/core";
+import { ReactNode } from "react";
+
+import { useComponentsContext } from "../../../../editor/ComponentsContext.js";
+import { useBlockNoteEditor } from "../../../../hooks/useBlockNoteEditor.js";
+import { DragHandleMenuProps } from "../DragHandleMenuProps.js";
+
+export const TableRowHeaderItem = <
```

Comment:
2 things:

1. Imo would be nice to also have this on the 1st row table handle (same for column headers)
2. Not specific to this component, but menu items in general look wonky with ShadCN

Reply:
1. Good idea
2. What specifically is wonky? I'm not familiar with it enough to know what feels off about it.

Reply:
For 2: sorry, I should specify - menu items with a checkmark show a space for it even when unchecked:
![Screenshot 2025-02-28 at 10 51 55](https://github.com/user-attachments/assets/392672e6-2c7e-42eb-92fb-9aa054e15bb3)
![Screenshot 2025-02-28 at 10 52 08](https://github.com/user-attachments/assets/ef720c75-1cb0-45b6-9788-a515ad80279b)